src/pkg/export/export.go

```diff
@@ -0,0 +1,45 @@
+package export
+
+import (
+	"context"
+	"io"
+)
+
+// Collection is the interface that is returned to the SDK consumer
+type Collection interface {
+	// GetBasePath gets the base path of the collection
+	GetBasePath() string
+
+	// GetItems gets the items within the collection(folder)
+	GetItems(context.Context) <-chan Item
```

Comment:
no strong opinions in this case. I have found that having functions named the same across different interfaces can make using tools like grep/GitHub's code explorer more difficult because they don't always take into account type information. In this case though I'm not sure what a better name would be and I have no strong opinion on including `Get` in function names

Reply:
Hmm, I was kinda on the fence about it. I've renamed `GetBasePath` to `BasePath` and `GetItems` to `Items`.